app/org/sagebionetworks/bridge/services/UploadService.java

```diff
@@ -148,7 +157,8 @@ public UploadSession createUpload(StudyIdentifier studyId, StudyParticipant part
             uploadId = originalUploadId;
         } else {
             // This is a new upload.
-            Upload upload = uploadDao.createUpload(uploadRequest, studyId, participant.getHealthCode());
+            Upload upload = uploadDao.createUpload(uploadRequest, studyId, participant.getHealthCode(),
+                    originalUploadId);
             uploadId = upload.getUploadId();
 
             if (originalUploadId != null) {
```

Comment:
Is there a reason we shouldn't set UploadStatus.DUPLICATE here, and wait for upload complete? 

If we set it in upload complete, could we call it UploadStatus.VALIDATION_SKIPPED (assigned on upload complete) to distinguish this from duplicate detection (assigned during upload creation)? Right now looking at the Upload object would confuse me to see one with with a non-null duplicateUploadId yet not be marked UploadStatus.DUPLICATE, and a rename would keep meaning of the UploadStatuses independent of whether it's a duplicate